src/datapump/sync/sync.py

```diff
@@ -723,6 +723,122 @@ def get_latest_release(self) -> Tuple[str, List[str]]:
         return latest_release, source_uri
 
 
+class DISTAlertsSync(Sync):
+    """
+    Defines jobs to create new DIST alerts assets once a new release is available.
+    """
+
+    dataset = "umd_dist_alerts"
+    source_bucket = "earthenginepartners-hansen"
+    source_prefix = "DIST-ALERT"
+    input_calc = """
+        np.where((A>=30) & (A<255) & (B>0) & (C>=2) & (C<255),
+            np.where(C<4, 20000 + B, 30000 + B),
+            0        
+        )
+    """
+
+    def __init__(self, sync_version: str):
+        super().__init__(sync_version)
+
+    def get_latest_release(self) -> Tuple[str, List[str]]:
+        """
+        Get the version of the latest *complete* release in GCS
+        """
+
+        # Raw tiles are just updated in-place
+        source_uris = [
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-ANOM-MAX",
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-DIST-DATE",
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-DIST-COUNT",
+        ]
+
+        # This file is updated once tiles are updated
+        upload_date_text = get_gs_file_as_text(
+            self.source_bucket, f"{self.source_prefix}/uploadDate.txt"
+        )
+
+        # Example string: "Updated Fri Apr 15 14:27:01 2022 UTC"
+        upload_date = upload_date_text[12:-5]
+        LOGGER.info(f"Last DIST-Alert upload date: {upload_date}")
+        latest_release = datetime.strptime(upload_date, "%b %d %H:%M:%S %Y").strftime(
+            "v%Y%m%d"
+        )
+
+        return latest_release, source_uris
+    
+    def build_jobs (self, config: DatapumpConfig) -> List[Job]:
+        latest_api_version = self.get_latest_api_version(self.dataset_name)
```

Comment:
self.dataset_name is inconsistent with line 731, probably rename dataset to dataset_name at line 731 to be consistent with the other alerts.